app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -522,6 +542,11 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
     $scope.angularForm[$scope.authType + '_auth_status'].$setViewValue(updatedValue);
   };
 
+  $scope.updateHawkularHostname = function(value) {
+    $scope.angularForm['hawkular_hostname'].$setViewValue(value);
+    $('#hawkular_hostname').val(value);
```

Comment:
> no reason to access it magically

Sure, if just the assignment `$scope.emsCommonModel.hawkular_hostname = value` works, why not?